src/Network/NWParameters.cs

```diff
@@ -234,15 +234,10 @@ public NWParameters Clone ()
 		[DllImport (Constants.NetworkLibrary)]
 		static extern void nw_parameters_require_interface (nw_parameters_t parameters, IntPtr handleInterface);
 
-		public void RequireInterface (NWInterface iface)
-		{
-			nw_parameters_require_interface (GetCheckedHandle (), iface.GetHandle ());
-		}
-
 		[DllImport (Constants.NetworkLibrary)]
 		static extern IntPtr nw_parameters_copy_required_interface (nw_parameters_t parameters);
 
-		public NWInterface Interface {
+		public NWInterface RequireInterface {
```

Comment:
I think `RequiredInterface` sounds better (mostly because `Require` is a verb, and this is a property, not a method).

Reply:
True, that is a typo, the C API uses required.